modules/deb.sh

```diff
@@ -221,6 +221,45 @@ function deb_get_and_install_package() {
     deb_install_package "work/deb-packages/${DEB_NAME}" "work/rootfs/"
 }
 
+#  Install binary files of a deb package into work/deb-packages from a local directory
+#  $1 - Directory path
+#
+function deb_get_and_install_input_package() {
+    if ! [[ $# -eq 1 ]]; then
+        log_fatal "deb_get_and_install_input_package() requires 1 argument"
+    fi
+
+    local -r package_path="$1"
+    local -r package_order_file="${package_path}/package_order"
+    local -a order=()
+
+    # Install packages by set order if file exists
+    if [ -e "${package_order_file}" ]; then
```

Comment:
I would check for regular file instead
```suggestion
    if [ -f "${package_order_file}" ]; then
```

Otherwise you risk to break if is a directory for example